build.gradle

```diff
@@ -54,17 +55,17 @@ compileJava {
 }
 
 dependencies {
-    compile('org.seleniumhq.selenium:selenium-java:3.4.0'){
+    compile('org.seleniumhq.selenium:selenium-java:3.5.1'){
```

Comment:
Probably, we could also define selenium dependency with `forced` flag, to make sure it is always used for our lib, even if Selenium component version is updated for the main project?

https://docs.gradle.org/3.3/dsl/org.gradle.api.artifacts.dsl.DependencyHandler.html